src/main/java/TaskList.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import java.util.ArrayList;
+import duke.task.Task;
+
+/**
+ * Represents a list containing all the tasks.
+ * A TaskList contains an ArrayList of the tasks.
+ */
+public class TaskList {
+
+    private ArrayList<Task> tasks;
+
+    /**
+     * Creates a TaskList Object containing an empty ArrayList.
+     */
+    public TaskList() {
+        this(new ArrayList<Task>());
+    }
+
+    /**
+     * Creates a TaskList Object containing the ArrayList of tasks given.
+     * @param tasks ArrayList of tasks to be stored in TaskList.
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Deletes a task from the TaskList stored in the index given.
+     * @param index index of the task to be deleted.
+     */
+    public void delete(int index) {
+        tasks.remove(index);
```

Comment:
I believe this will throw an exception if the user specifies an index that is beyond the length of the ArrayList. May want to try it out by writing a test case for it.